src/qutip_qip/decompose/_utility.py

```diff
@@ -0,0 +1,31 @@
+from qutip import Qobj
+
+
+class MethodError(Exception):
+    """When invalid method is chosen, this error is raised."""
+
+    pass
+
+
+def check_gate(gate, num_qubits):
+    """Verifies input is a valid quantum gate.
+
+    Parameters
+    ----------
+    gate : :class:`qutip.Qobj`
```

Comment:
remove qutip - only :class:`.Qobj`

Reply:
Is there a reason?I'm not sure but there seems to be a way to link  `:class:`qutip.Qobj` to the qutip doc, with some extension. I tried it once but didn't work. That is still on my to-do list.

Reply:
I think I need to use `:class:~qutip.Qobj` but I will double check soon.